bot/cogs/alias.py

```diff
@@ -5,6 +5,7 @@
 from discord import Colour, Embed, Member, User
 from discord.ext.commands import Bot, Cog, Command, Context, clean_content, command, group
 
+from bot.cogs.extensions import Extension
```

Comment:
The converter uses the `EXTENSIONS` set, which is a module-level constant in `extensions.py`. Importing it here means it iterates the modules again, but the alias must use the converter... Is there a good way to avoid that happening or is it not worth the trouble?

Reply:
As far as I can tell, it will not iterate twice by importing it here, since it has already been imported when we load the `alias` cog and Python will not import it again. I've added a temporary logging call and that seems to confirm that the code is just executed once. It's only when I reload the `extensions` extension that the logging call is made again, which is to be expected, since `discord.py` explicitly makes sure to re-import the module to refresh the code.

Reply:
I suspected this may be the cas. I just don't quite understand how imports work I suppose. Thanks for figuring that out though.